examples/layers.ipynb

```diff
@@ -456,8 +462,9 @@
     "\n",
     "data = workflow.get_dataframe(\n",
     "    ge.QueryRectangle(\n",
-    "        [-111.533203125, -4.482421875, 114.345703125, 73.388671875],\n",
-    "        [time, time]\n",
+    "        ge.BoundingBox2D(-111.533203125, -4.482421875, 114.345703125, 73.388671875),\n",
+    "        ge.TimeInterval(time, time),\n",
+    "        ge.SpatialResolution(0.1, 0.1)\n",
     "        )\n",
```

Comment:
This is quite verbose. Did you intend to do it this way?

Otherwise, you could call it sill with tuples and then use the constructor within `QueryRectangle` depending on the type.

Reply:
yes i think we should do it this way (for now).
We can add tuples as params in a new PR. There we need to do some kind of "into" conversion and a lot of asserts.